src/tools/prepareTodos.tsx

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../types/Todo';
+import { FilterField } from './constants';
+
+export const prepareTodos = (
+  readyTodos: Todo[],
+  title: string,
+  filterField: string,
+) => {
+  const filteredTodos = readyTodos.filter((todo: Todo) =>
+    todo.title.toLowerCase().includes(title.toLowerCase()),
+  );
+
+  if (filterField) {
+    switch (filterField) {
+      case FilterField.ALL:
+        return filteredTodos;
+
+      case FilterField.ACTIVE:
+        return filteredTodos.filter(todo => !todo.completed);
+
+      case FilterField.COMPLETED:
+        return filteredTodos.filter(todo => todo.completed);
+
+      default:
+        return filteredTodos;
+    }
+  }
+
+  return filteredTodos;
+};
```

Comment:
```suggestion
export const prepareTodos = (
  readyTodos: Todo[],
  title: string,
  filterField: FilterField,
) => {
  return readyTodos.filter((todo: Todo) => {
     const titleCondition = todo.title
            .toLowerCase()
            .includes(title.toLowerCase());
            
     switch (filterField) {
       case FilterField.All:
         return firstCondition;
         
       case  FilterField.Active:
         return firstCondition && !todo.completed;
         
       case FilterField.Completed:
         return firstCondition && todo.completed;
         
       default:
         return true;
  });
};
```

Reply:
As I mentioned in previous review you can use 1 filter for getting the same result. Please fix it.